src/CMakeLists.txt

```diff
@@ -456,6 +456,14 @@ find_package(PUGIXML REQUIRED)
 find_package(HUNSPELL REQUIRED)
 find_package(Boost 1.44)
 
+
+# FreeBSD's sysinfo() call is done in a package, libsysinfo.
+# Find the library and include it appropriately.
+
+find_library(SYSINFO_LIBRARY NAMES sysinfo)
```

Comment:
Will this work on all OS's Mudlet supports or should this be guarded for FreeBSD only?